index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="index.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+  <meta charset="utf-8">
+</head>
+<body>
+  <h1 class="calculator-title">CALCULATOR</h1>
+  <div class="calculator-calcBody">
+    <div type="text" class="calculator-valueDisplay">
+      <script type="text/JavaScript" src="index.js"></script>
+      <script>displayValue()</script>
+    </div>
+    <button class="calculator-row calculator-button-floating-box">AC</button>
```

Comment:
you have both `calculator-row` and `calculator-button-floating-box` on all of these buttons. How are buttons also rows when there are multiple buttons per row?

Also consider changing `calculator-button-floating-box` to just `calculator-button`